C++/libs/Tuprules.tup

```diff
@@ -3,7 +3,7 @@
 CC = g++
 
 # Uncomment to build with debug symbols
-#CFLAGS += -g
+CFLAGS += -g
```

Comment:
This doesn't really bother me, but I'm hoping [Hatch](https://github.com/CachedNerds/Hatch) will help us with managing debug/release builds better.

Reply:
Thank you for noticing this, this was simply something I forgot to disable after a debugging session.  I agree completely about Hatch.